docs/Install/Running_Swarm_Learning_examples_using_SLM-UI.md

```diff
@@ -0,0 +1,38 @@
+# <a name="GUID-A2B92980-7281-4B0A-989F-33097B7C96A5/"> Running Swarm Learning examples using SLM-UI
+
+**TIP:**
+
+Before you start using the SLM-UI, HPE recommends you to read the *HPE Swarm Learning User Guide* to understand about the architecture of Swarm Learning, how these nodes work, how model training happens, and Swarm Learning Concepts section.
```

Comment:
Reference to user guide to be dropped and instead use URL to main landing page#Documentation section